### PR TITLE
generating SDK and adding SDK test for patch status instance view

### PR DESCRIPTION
**Pull request by jemex**

### CODE REVIEW

**Comment by jemex** [comment]
This is related swagger change https://github.com/Azure/azure-rest-api-specs/pull/10216

**Comment by isra-fel** [comment]
Hi @jemex please regenerate once swagger PR is merged, and you might want to update the version for release.

**Comment by jemex** [comment]
> Hi @jemex please regenerate once swagger PR is merged, and you might want to update the version for release.

Swagger is merged and the SDK is generated from the master. Version release is updated.

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj [hunk 1]
```diff
@@ -14,7 +14,9 @@
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This is a public release of the Azure Compute SDK. Included with this release is the vmHealth property added to the VMInstanceView API.
```
Comment (isra-fel) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj:17]: The version on line 12 of this file needs to be updated.

Comment (jemex) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj:17]: > The version on line 12 of this file needs to be updated.

The version is updated now

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**